terraform/rds.tf

```diff
@@ -80,6 +80,7 @@ resource "aws_rds_cluster_instance" "ehr_out_service_db_instances" {
   }
 }
 
+# TODO: Remove as part of PRMT-4706:
```

Comment:
Just a reminder here, checked the board and there seems to be no description on `PRMT-4706`, could we please add one?